addins/Cake.SquareLogo.yml

```diff
@@ -0,0 +1,9 @@
+Name: Cake.SquareLogo
+NuGet: Cake.SquareLogo
+Assemblies:
+- "/**/Cake.SquareLogo.dll"
+Repository: https://www.nuget.org/packages/Cake.SquareLogo/
+Author: Cake.SquareLogo
+Description: "Package Description"
```

Comment:
IMHO addins like this without any useful metadata should be filtered

Reply:
How can I detect whether the metadata is "useful" or not?

Reply:
How about having a manual blacklist were addins which didn't pass PR review were added.

Reply:
Here's what I will do: https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/28